docs/powervs-faq.md

```diff
@@ -0,0 +1,24 @@
+# PowerVS Frequently Asked Questions
+
+---
+## When Building out a Cluster
+
+#### VM Console shows "grub>"
+Not the end of the world. Just ESC and ENTER, as the message on screen instructs.
```

Comment:
We need to be very clear here given that there is multiple grub> and grub rescue>, which are caused by different issues. I will share a text to be added, we need to ensure we give the user steps to report the issue so we can have it formally documented.

Reply:
@miyamotoh ^

Reply:
Good point about the rescue prompt. Maybe we explicitly say "not grub rescue>" or add a separate bullet for "grub rescue>". 

Setting up an interactive channel is not a mission for this doc site. It would certainly be happy to point to it, when available, though.

Reply:
grub>: the grub prompt on a blank screen. GRUB 2 has found the boot information but has been either unable to locate or unable to use an existing GRUB 2 configuration file (usually grub.cfg). 

grub rescue>: the rescue mode. GRUB 2 is unable to find the grub folder or its contents are missing/corrupted. The grub folder contains the GRUB 2 menu, modules and stored environmental data. 

Reply:
@miyamotoh we can if the use gets a grub>, we can recommend the execution of the command normal (grub> normal), which will resume the normal boot. If the uses gets a grub rescue>, then the solution/workaround will depend on the issue.

Reply:
Feedback incorporated in the latest push.